src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml

```diff
@@ -5,6 +5,19 @@
 @using EA.Weee.Web.ViewModels.Shared
 @model EA.Weee.Web.ViewModels.Shared.ViewTransferNoteViewModel
 
+<!-- screen 6.09a: View Void Transfer History -->
+@if (Model.DisplayEvidenceNoteHistoryData)
+{
+    <div class="govuk-width-container govuk-!-padding-left-0">
+        @(this.WeeeGds().CreateProgressiveDisclosure("Transfer history",
+                        @Html.Partial("~/Views/Shared/_EvidenceTransferProgressiveDisclosurePartial.cshtml").ToString(),
+                        "Hidden content",
+                        "Progressive disclosure",
```

Comment:
Can be removed a transfer note wont have the history its the original evidence note that has the history. George to update the wireframe.

Reply:
done